assets/js/checkout.js

```diff
@@ -27,7 +27,7 @@ $(function () {
   switch ($.query.get('addon')) {
     case 'subscription-type': _updateCheckout({ subscription_type: true }); break;
     case 'setup-deployment': _updateCheckout({ setup_deployment: true }); break;
-    case 'dedicated-support': _updateCheckout({ dedicated_support: true }); break;
+    case 'pro-support': _updateCheckout({ pro_support: true }); break;
```

Comment:
Did you remember to update the incoming "getting started" links with this change?

Reply:
I did indeed!